packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import * as _ from 'lodash'
+
+import { getNextElement, focusAsync } from './focusUtilities'
+
+import { AutoFocusZoneProps } from './AutoFocusZone.types'
+import getUnhandledProps from '../../getUnhandledProps'
+import getElementType from '../../getElementType'
+import * as customPropTypes from '../../customPropTypes'
+import callable from '../../callable'
+import Ref from '../../../components/Ref/Ref'
+
+/** AutoFocusZone is used to focus the first element inside it's children. */
```

Comment:
Not only first, but als by selector
...`is used to focus inner element on mount`...

Reply:
Changed to: `AutoFocusZone is used to focus inner element on mount.` You are right, the props description will indicate the details.